sdk/storage/ci.yml

```diff
@@ -33,6 +33,7 @@ extends:
       LiveTestCtestRegex: azure-storage
       Clouds: Preview
       SupportedClouds: Preview
+      UseFederatedAuth: false
```

Comment:
what does this do?

Reply:
This switch tells the pipeline if it should use the FederatedAuth (azurepipelinecrdential). Right now storage is NOT ready to use that model so storage is being opted out.